buildSrc/src/main/kotlin/io/spine/internal/gradle/publish/SpinePublishing.kt

```diff
@@ -116,9 +116,11 @@ fun Project.spinePublishing(configuration: SpinePublishing.() -> Unit) {
 /**
  * A Gradle extension for setting up publishing of spine modules using `maven-publish` plugin.
  *
+ * @param targetProject a project in which the extension is opened
```

Comment:
And I don't understand what you mean. Please speak in terms of publishing. Is this the project which I want to publish, or the one that has child projects to be published? If so, then please describe accordingly.